tuf/builder.go

```diff
@@ -242,11 +246,16 @@ func (rb *repoBuilder) GetConsistentInfo(roleName string) ConsistentInfo {
 }
 
 func (rb *repoBuilder) Load(roleName string, content []byte, minVersion int, allowExpired bool) error {
+	return rb.LoadOptions(roleName, content, minVersion, allowExpired, false, false)
+}
+
+// LoadOptions adds additional flags that should only be used for updating the root.json
+func (rb *repoBuilder) LoadOptions(roleName string, content []byte, minVersion int, allowExpired, skipChecksum, allowLoaded bool) error {
```

Comment:
That makes sense, I did the simple refactor for now and I think a larger refactor of the builders should happen in a future PR.